core/services/keeper/models.go

```diff
@@ -13,12 +17,12 @@ type Registry struct {
 	NumKeepers        int32
 }
 type UpkeepRegistration struct {
-	ID                  int32
-	CheckData           []byte
-	ExecuteGas          uint64
-	LastRunBlockHeight  int64
-	RegistryID          int64
-	Registry            Registry
-	UpkeepID            int64
-	PositioningConstant int32
+	ID                 int32
+	CheckData          []byte
+	ExecuteGas         uint64
+	LastRunBlockHeight int64
+	RegistryID         int64
+	Registry           Registry
+	UpkeepID           int64
+	LastKeeperIndex    sql.NullInt64
```

Comment:
I don't think this type supports json marshaling - if we ever export this data to the frontend or CLI or something then this type will have to change. Doesn't need to be a blocker, but I think we can future-proof our code a bit more by using types that support marshaling. Ex every other NullInt uses our internal `core/null/intX.go`.

Reply:
alright will switch it